ads-on-read-the-docs.rst

```diff
@@ -0,0 +1,61 @@
+.. post:: March 9, 2016
+   :tags: ads, status, sustainability
+
+Advertising on Read the Docs Community Sites
+============================================
+
+On Monday we started adding a small ad on readthedocs.org pages.
```

Comment:
* "On Monday" -> "On Monday,"
* "started adding" -> "added"

Might be worth mentioning it's the second iteration of this, maybe?